index.html

```diff
@@ -0,0 +1,314 @@
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div id="contact" class="form-container">
+        <div class="form-text-container">
+            <h2 class="text-secondary-dark">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+        <form id="contact_form">
+          <label hidden for="full-name"> Full Name </label>
+          <input type="text" name="full-name" id="full-name" placeholder="Full name" required>
+          <label hidden for="email"> Email address </label>
+          <input type="text" name="email" id="email" placeholder="Email address" required> 
+          <label hidden for="message"> Message </label>
+          <textarea name="message" id="message"></textarea>
+          <button type="submit" class="btn-form">Get in touch</button>
+          <!-- <div class="error_msg"></div> -->
```

Comment:
- Kindly remove this commented HTML code as it makes your project look unprofessional. 👮‍♂️